docs/kyma/docs/05-01-overview.md

```diff
@@ -0,0 +1,63 @@
+
+> **NOTE:** The installation and configuration templates serve as the basis for creating the installation (`kyma-installer-local.yaml` and `kyma-installer-cluster.yaml`) and configuration (`kyma-config-local.yaml` and `kyma-config-cluster.yaml`) release artefacts.
+
+Both configuration files contain pre-defined overrides in the form of Secrets and ConfigMaps that the Installer uses during the installation to replace default values specified in `values.yaml` files or to provide required configuration values. While the local configuration file contains hardcoded override values, the cluster configuration file is based on placeholders that are replaced with real values during the installation process.
+You can add multiple Secrets and ConfigMaps for one component. If you duplicate the same parameter in a few Secrets or ConfigMaps for a given component but give it different values, the Installer accepts the last value in the file.
+
+>**CAUTION:** Both `values.yaml` and the configuration files contain production settings that are tested and recommended. Note that you modify them on your own risk.
+
+## Installation configuration
+
+Before you start the Kyma installation process, you can customize the default settings in the installation and configuration files.
+
+### Components
+
+One of the installation artifacts is the Kyma Installer image that contains all component charts located in the `kyma/resources` folder. Both cluster and local installation files contain a full list of these component names and their Namespaces.
+Components that are not an integral part of the default Kyma Lite package, are preceded with hashtags (#) which means that the Installer skips them during the installation process.   
+You can customize the component installation files by:
+- Removing hashtags in front of the component entries to enable a component installation.
+- Adding hashtags to disable a component installation.
+- Adding new components to the list along with their chart definition in the `kyma/resources` folder. In that case you must create your own [Kyma Installer image](#configuration-use-your-own-kyma-installer-image) as you are adding new chart configuration.
+
+For more details on custom component installation, see [this](#configuration-custom-component-installation) document.
+
+### Overrides
+
+The configuration files provide production-ready settings for the Installer. These include ConfigMaps and Secrets with the overrides for the values hardcoded in the `values.yaml` files of component charts.
+
+You can modify local settings, such as memory limits for a given resource, by changing values in the existing ConfigMaps and Secrets or adding new ones. Every ConfigMap and Secret that the Installer reads must contain an `installer: overrides` label and a `component: {component-name}` label if it refers to a specific component. Add the `kyma-project.io/installation: ""` label to all non-Kubernetes resources to delete or search for them easier.
```

Comment:
you have already 3 label, it make sense to split them into 3 bullet points


Every ConfigMap and Secret that the Installer reads must contain the following labels:
* `installer: overrides`
* `component: {COMPONENT_NAME}` if resource refers to a specific component
* `kyma-project.io/installation: ""` to all non-Kubernetes resources to delete or search for them easier.



btw, tbh I don't understand `to delete or search for them easier.`

Reply:
I created the bullet points and improved the explanation of the `kyma-project.io/installation: ""` label